src/components/TabBar.tsx

```diff
@@ -148,6 +174,7 @@ const TabBar = ({
         onLayout={handleLayout(o.value)}
         onPress={handlePress(o.value)}
         hitSlop={hitSlop}
+        iconPosition={o.iconPosition}
```

Comment:
doesn't the spread of `o` already do this?

Reply:
Done